lib/tab-stop-list.js

```diff
@@ -0,0 +1,46 @@
+/** @babel */
```

Comment:
It would be nice to avoid using Babel on new files. Everything except the fancy `import` syntax works fine.

Reply:
Cool, was unclear on what the best practice was for new files.